lib/fluent/plugin/storage_local.rb

```diff
@@ -56,7 +56,7 @@ def configure(conf)
         elsif @path
           # ok
         else # @_plugin_id_configured is true
-          raise NotImplementedError, "implement this feature later with system_config"
+          raise NotImplementedError, "implement this feature later with system_config. Add <storage> section for file store or remove @id for memory store temporary"
```

Comment:
Yeah, I know. The problem is users now hard to use <storage> without file because storage plugin doesn't have store type option.
Currently, existing users hit this error because they use following configuration for v0.12.

```
<source>
  @type dummy
  @id input_dummy
  # ...
</source>
```

I think this configuration should work with v0.14 without error. My change is temporary. Better way is fallback to on memory store, not raising `NotImplementedError`.